### PR TITLE
Add dependabot config

### PR DESCRIPTION
**Pull request by Lauraducky**

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - "jimcullenaus"
+      - "lauraducky"
+      - "arthurs"
+      - "evangelinexx"
+      - "drewfreyling"
+      - "gillianmilne"
+      - "chromadream"
+    commit_message:
+      prefix: "fix"
+      prefix_development: "chore"
```
Comment (jimcullenaus) [inline .dependabot/config.yml]: what's this?

File: .dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "weekly"
```
Comment (jimcullenaus) [inline .dependabot/config.yml]: yes pls

File: .dependabot/config.yml (hunk 1 above)
Comment (Lauraducky) [reply .dependabot/config.yml]: if it's a dev dependency, it will look like `chore(deps-dev): description` otherwise `fix(deps): description`

**Pull request merged**